src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Button from './ButtonSkeleton';
+
+const operators = ['÷', 'x', '-', '+', '='];
+const digits = ['AC', '+/-', '%', '7', '8', '9', '4', '5', '6', '1', '2', '3', '0', '.'];
```

Comment:
- [ ] [optional] It would be nicer if you created your variables inside the calculator class, just after your constructor.